### DISCUSSION TITLE
health/damage/respawn

### DISCUSSION THREAD

**Discussion by Omegarte** [samolego/Taterzens#79] [Q&A] [answered]
This mod is great. 

A way to adjust the amount of health an NPC has and the amount of damage it does (with its bare hand) would be greatly appreciated.  

Someway to automatically respawn after X amount of time after death would also be a useful features.

Thanks again and great work.

**Comment by samolego** [comment, marked as answer]
Hi, the damage & health thingy is already possible in vanilla :).

400 health, 6 attack dmg:
```brigadier
/data merge entity <taterzen uuid> {Health:400f,Attributes:[{Name:generic.max_health,Base:400},{Name:generic.attack_damage,Base:6}]}
```

**Comment by Omegarte** [comment]
Great, thanks!
